### ISSUE TITLE
Crash with Xamarin 3.11.590 in release mode

### ISSUE THREAD

**Issue by Ljusnan** [Redth/ZXing.Net.Mobile#172]
The sample app crashes with Xamarin stable version 3.11.590 in release mode. I downloaded ZXing  version 1.4.7.2 in Xamarins Component Store: https://components.xamarin.com/gettingstarted/zxing.net.mobile

**Comment by Redth** [comment]
Could you provide more information?

What type of project are you running, a Unified API, or Classic API?  Also please describe how and when it's crashing and how to reproduce the crash.

Finally, a stack trace would be helpful too.

**Comment by Ljusnan** [comment]
I downloaded ZXing from the Xamarins Component Store and started the sample app in release mode.  I changed nothing else. The app crashed right away at the start. I used a Samsung Galaxy S4 Mini, but I think it has nothing to do with the device. A coworker and a client tested it both on different devices with same crash.
Sorry, there is no more information I could give you. There is no stack trace in Visual Studio since its running in release mode.
Xamarin version is 3.11.590, ZXing version is 1.4.7.2 and Visual Studio version is 12.0.31101.00 Update 4.
